### PR TITLE
Allow to change OcCollector Service name via LINKERD2_PROXY_TRACING_S…

### PR DESCRIPTION
**Pull request by GrigoriyMikhalkin**
Solves https://github.com/linkerd/linkerd2/issues/11157

### CODE REVIEW

**Review by adleong — APPROVED** [review]

File: linkerd/app/src/env.rs [hunk 1]
```diff
@@ -249,6 +249,8 @@ const ENV_OUTBOUND_HTTP1_CONNECTION_POOL_IDLE_TIMEOUT: &str =
 
 const ENV_SHUTDOWN_GRACE_PERIOD: &str = "LINKERD2_PROXY_SHUTDOWN_GRACE_PERIOD";
 
+const ENV_PROXY_TRACING_SERVICE_NAME: &str = "LINKERD2_PROXY_TRACING_SERVICE_NAME";
```
Comment (olix0r) [inline linkerd/app/src/env.rs:252]: For the sake of consistency, we already have:
```
pub const ENV_TRACE_ATTRIBUTES_PATH: &str = "LINKERD2_PROXY_TRACE_ATTRIBUTES_PATH";
```

can we call this `LINKERD2_PROXY_TRACE_SERVICE_NAME` and group the definitions?

Comment (GrigoriyMikhalkin) [reply linkerd/app/src/env.rs]: Done

**Comment by GrigoriyMikhalkin** [comment]
@alpeb Do i understand it correctly that it was already implemented(by a different approach) in https://github.com/linkerd/linkerd2/pull/13130? So this PR can be closed, i assume?

**Comment by sfleen** [comment]
Correct, we should be able to close this PR now.

**Pull request closed without merging**